src/Playwright/Transport/Connection.cs

```diff
@@ -136,7 +136,9 @@ public void Dispose()
                         object obj = propertyDescriptor.GetValue(args);
                         if (obj != null)
                         {
+#pragma warning disable CA1845
```

Comment:
CA1845 is not in the `Playwright.ruleset` anywhere so I'm just ignoring.

Reply:
These are probably new built-in default rules from the SDK for the .NET 6 analysis level.